### DISCUSSION TITLE
Differences with the call control native option

### DISCUSSION THREAD

**Discussion by mathieu-mp** [balansse/google-meet-jabra#21] [General]
Hi,

Could you explain what are the differences between using this extension and using the native "call control" option?

![image](https://github.com/user-attachments/assets/e7187e40-e48b-4cc9-8ab3-eb75d5fb032f)

Thanks!

**Comment by balansse** [comment]
I wrote this extension before Google Meet added native support. Honestly speaking, now that it is supported natively, this extension has become obsolete.
